flink-table/flink-sql-parser/src/main/codegen/includes/parserImpls.ftl

```diff
@@ -55,6 +55,30 @@ SqlUseCatalog SqlUseCatalog() :
     }
 }
 
+/**
+* Parses a create catalog statement.
+* CREATE CATALOG catalog_name [WITH (property_name=property_value, ...)];
+*/
+SqlCreate SqlCreateCatalog(Span s, boolean replace) :
+{
+    SqlParserPos startPos;
+    SqlIdentifier catalogName;
+    SqlNodeList propertyList = SqlNodeList.EMPTY;
+}
+{
+    <CATALOG> { startPos = getPos(); }
+    catalogName = CompoundIdentifier()
```

Comment:
Why do we support CompoundIdentifier for catalog names?

Reply:
should be simple identifier. Fixed